noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/AsterixHttpRequest.scala

```diff
@@ -0,0 +1,41 @@
+package edu.uci.ics.cloudberry.noah.feed
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.libs.ws.ahc.AhcWSClient
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object AsterixHttpRequest {
+
+  def createClient(): AhcWSClient = {
+    implicit val system = ActorSystem("my-system")
+    implicit val materializer = ActorMaterializer()
+
+    val wsClient = AhcWSClient()
+    return wsClient
```

Comment:
every result of the last statement will be the return value.  so these two lines could write as 
```
AhcWSClient()
```